packages/foreman/foreman/foreman.spec

```diff
@@ -759,6 +762,33 @@ install -Dp -m0644 %{SOURCE5} %{buildroot}%{_tmpfilesdir}/%{name}.conf
 install -Dp -m0644 extras/systemd/%{name}.service %{buildroot}%{_unitdir}/%{name}.service
 install -Dp -m0644 extras/systemd/%{name}.socket %{buildroot}%{_unitdir}/%{name}.socket
 
+# SELinux libexec wrappers
+%if 0%{?scl:1}
+  cat > %{buildroot}%{_libexecdir}/%{name}/ruby-selinux <<-EOF
+  #!/bin/bash
+  # Shell wrapper with SELinux transition into foreman_rails_t domain.
+  source scl_source enable %{scl_prefix}
+  exec ruby "\$@"
+EOF
+  cat > %{buildroot}%{_libexecdir}/%{name}/sidekiq-selinux <<-EOF
+  #!/bin/bash
+  # Shell wrapper with SELinux transition into foreman_rails_t domain.
+  source scl_source enable %{scl_prefix}
```

Comment:
Looks like `%{scl_prefix}` contains a trailing dash, `%{scl}` should be just the name
```
systemd[1]: Started Foreman jobs daemon - orchestrator on sidekiq.
dynflow-sidekiq@orchestrator[2898]: Can't read /etc/scl/conf/tfm-, tfm- is probably not installed.
systemd[1]: dynflow-sidekiq@orchestrator.service: main process exited, code=exited, status=127/n/a
systemd[1]: Unit dynflow-sidekiq@orchestrator.service entered failed state.
systemd[1]: dynflow-sidekiq@orchestrator.service failed.
```

Reply:
Whoops, thanks for testing. I already have a VM with all manual patching this and I'd need to spin new nightly for this as I don't want to break this one :-)